src/backend/marsha/settings.py

```diff
@@ -77,9 +77,13 @@ class Base(Configuration):
     SESSION_COOKIE_SECURE = True
     SECURE_CONTENT_TYPE_NOSNIFF = True
     CSRF_COOKIE_SECURE = True
-    SILENCED_SYSTEM_CHECKS = [
-        "security.W019"  # Disable because we want to use SAMEORIGIN as value for X_FRAME_OPTIONS
-    ]
+    SILENCED_SYSTEM_CHECKS = values.ListValue(
+        [
+            "security.W019",  # We want to use SAMEORIGIN as value for X_FRAME_OPTIONS
+            "security.W008",  # Disable SECURE_SSL_REDIRECT check, not usable with a wsgi server.
```

Comment:
Actually this check could be done by default and disabled voluntarily in Arnold because we know that in this case the https termination is done in nginx... ?

Reply:
So in that case I just put:

```
SILENCED_SYSTEM_CHECKS = values.ListValue([])
```

And then in Arnold I set the value we want in an environment variable ?

Reply:
probably yes.

BTW clickjacking should only be disabled on the LTI view anyway not on future pages that we might do.

Reply:
clickjacking is deactivated on the LTI view and used on all other routes with `SAMEORIGIN` value.

Reply:
Why don't you set it to `DENY`? If I'm not mistaken, `SAMEORIGIN` is only required for the development view which is anyway disabled in production.

Reply:
Yes, this middleware is not configured as expected... Everything can be `DENY` except the LTI view...